packages/mobile/src/screens/favorites-screen/FilterInput.tsx

```diff
@@ -0,0 +1,40 @@
+import IconFilter from 'app/assets/images/iconFilter.svg'
+import { SearchInput, Tile } from 'app/components/core'
+import { makeStyles } from 'app/styles'
+
+type FilterInputProps = {
+  value: string
+  onChange?: (newValue: string) => void
+  placeholder?: string
+}
+
+const useStyles = makeStyles(({ palette, spacing }) => ({
+  tileRoot: {
+    margin: spacing(4)
+  },
+  tile: {
+    padding: spacing(2)
+  },
+  input: {
+    backgroundColor: palette.neutralLight10
+  }
+}))
+
+export const FilterInput = ({
+  value,
+  onChange = () => {},
+  placeholder = ''
```

Comment:
and yeah similarly, i wouldnt worry about a deault value hre, just use searchinput defaults?